transport/src/commonMain/kotlin/com/genesys/cloud/messenger/transport/core/MessagingClientImpl.kt

```diff
@@ -364,3 +383,6 @@ internal class MessagingClientImpl(
         }
     }
 }
+
+private fun KProperty0<DeploymentConfig?>.isAutostartEnabled(): Boolean =
+    this.get()?.messenger?.apps?.conversations?.autoStart?.enabled == true
```

Comment:
Choo choo, null check 🚞🚃🚃🚃🚃🚋 coming through.

Reply:
Imagine that with if-not-null block.